services/UserAccountService.ts

```diff
@@ -33,6 +34,28 @@ export default class UserAccountService {
     return user;
   }
 
+  public async findByHandle(handle: string): Promise<UserModel> {
+    const user = await this.transactions.readOnly(async (txn) => Repositories
+      .user(txn)
+      .findByHandle(handle));
+    if (!user) throw new NotFoundError('User was not found');
+    return user;
+  }
+
+  /**
+   * Generate a default user handle in the format of "[firstName][lastName][6 digit has]"
+   * @param firstName User's first name
+   * @param lastName User's last name
+   * @returns default user handle
+   */
```

Comment:
nit: let's update the comment to add the dashes here as well. Also, I think the param and return doc comments aren't really necessary, and since we don't include them anywhere else, let's remove them.